.babelrc

```diff
@@ -1,17 +1,22 @@
 {
   "presets": [
     [
-      "env",
+      "@babel/preset-env",
       {
         "targets": {
-          "browsers": ["last 2 versions"],
-          "node": "current"
-        },
-        "forceAllTransforms": true
+          "node": "current",
```

Comment:
Can we use a [browserlist](https://github.com/browserslist/browserslist) query in a `. browserslistrc` (or under a `browserslist` key in `package.json`) instead of targets in `.babelrc`?

This is what `@babel/preset-env` doc recommends, as it's followed not only by Babel but other tools as well.

Also we should explicitly target the lowest `JSC` version Ledger Live Mobile has to deal with. For Android we ship the latest version, but on iOS we depend on what the OS offers.
AFAIK the lowest iOS version we support is the same as RN itself, so that would be 9.0.

And while we're at it, target our specific lowest Electron we use (unless we don't merge before migrating LLD?).

As a browserlist query that would be something akin to `electron 1.2, ios_saf 9, current node` (`ios_saf` is iOS Safari, which version follows the OS it's shipped with)